spec/schemas/ml._common.yaml

```diff
@@ -1214,10 +1214,20 @@ components:
           type: string
         type:
           type: string
+        description:
+          type: string
         parameters:
           $ref: '#/components/schemas/Parameters'
         include_output_in_agent_response:
           type: boolean
+        attributes:
+          type: object
+          properties:
+            input_schema:
+              type: string
+            strict:
+              type: boolean
+          additionalProperties: true
```

Comment:
This should be extracted to a separate schema named `ToolAttributes`, and replaced with a `$ref` here. Also add a `x-version-added: '3.0'`